### PR TITLE
Merge release/3.0 to release/4.0

### PR DESCRIPTION
**Pull request by marutosi**

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://marutosi-finn-release-4-0-merge.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: packaging/setup [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
```
Comment (linki) [inline packaging/setup:1]: why did you remove the shebang?

**Comment by tessi** [comment]
Hi @marutosi You are right, the change shall be put into release/4.0

We've just adressed this (and also updated both branches to use ruby 2.1.4).

Thanks and :+1: for seing this!
